library/pk.c

```diff
@@ -114,7 +114,7 @@ const mbedtls_pk_info_t *mbedtls_pk_info_from_type(mbedtls_pk_type_t pk_type)
         case MBEDTLS_PK_ECKEY_DH:
             return &mbedtls_eckeydh_info;
 #endif
-#if defined(MBEDTLS_ECDSA_C)
+#if defined(MBEDTLS_PK_CAN_ECDSA_SIGN) || defined(MBEDTLS_PK_CAN_ECDSA_VERIFY)
```

Comment:
Looks like this is going to be used a lot. Should we define shorthand `MBEDTLS_PK_CAN_ECDSA_SOME`? (In `library/pk_write.h` to avoid polluting the public API.) Wdyt?

Reply:
Note on commit structure (nothing to change here, just for the future): this commit is doing two distinct things (macro change, type change), I think two commits would have been preferable.